pkg/synthetictests/alerts.go

```diff
@@ -59,6 +61,20 @@ func testAlerts(events monitorapi.Intervals,
 		}
 	}
 
+	if kubeClient == nil {
+		kubeClient, err = framework.LoadClientset(true)
+		if err != nil {
+			panic(err)
+		}
+	}
+	nsExist, err := exutil.IsNamespaceExist(kubeClient, "openshift-monitoring")
+	if err != nil {
+		panic(err)
+	}
+	if !nsExist {
+		return []*junitapi.JUnitTestCase{}
```

Comment:
Better to avoid invoking `testAlerts` completely in https://github.com/openshift/origin/blob/master/pkg/synthetictests/event_junits.go#L54-L55. Question here to ask is whether there's a sufficient replacement for absence of the monitoring stack when checking the alerts in MicroShift. Does it make sense to e.g. run an external monitoring stack (or its simplified version)?

Reply:
>Better to avoid invoking testAlerts completely in https://github.com/openshift/origin/blob/master/pkg/synthetictests/event_junits.go#L54-L55. 

Based on the existence of  `"openshift-monitoring"`  ns right? So, IIUC the suggestion is to add this above check inside `event_junits.go` ?

>Question here to ask is whether there's a sufficient replacement for absence of the monitoring stack when checking the alerts in MicroShift. Does it make sense to e.g. run an external monitoring stack (or its simplified version)?

AFAIK there is no plan to add any monitoring stack in MicroShift as of now. So, we might want to skip them as well.